README.rst

```diff
@@ -795,7 +795,25 @@ file in the test project.
 For more information about the usage of celery in django, please refer to the
 `"First steps with Django" section in the celery documentation <https://docs.celeryproject.org/en/master/django/first-steps-with-django.html>`_.
 
-13. Import the automated tests
+13. Import Celery Tasks
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Add the following in your settings.py to import celery tasks from ``device_monitoring`` app.
+
+.. code-block:: python
+
+    CELERY_IMPORTS = ('openwisp_monitoring.device.tasks')
```

Comment:
This is a string and a tuple is expected. Either you add a trailing comma or convert it to a list

Reply:
I had added a `tuple` not sure if it occured due to `black`.